### ISSUE TITLE
Data set download

### ISSUE THREAD

**Issue by LJIAl** [drboog/Lafite#21]
Thank you very much for your work. Now I have the problem of data Set download failure. When downloading MS-COCO Training Set、MS-COCO Validation Set、LN-COCO Training Set and LN-COCO Testing Set,  I failed after downloading part of them.

**Comment by drboog** [comment]
I just tried and everything worked fine on my end, I guess maybe you met some issues like network connection (are you using VPN?).
